new files: fonts

ArsenijDKS opened the pull request:
new files: images and icons
	modified:   lib/main.dart

ArsenijDKS committed:
new files: fonts
	new files: images and icons
	modified:   lib/main.dart

Evgenia-bit commented on pubspec.yaml:
```diff
@@ -74,12 +75,13 @@ flutter:
   # "family" key with the font family name, and a "fonts" key with a
```
Желательно удалять лишние шаблонные комментарии

Evgenia-bit commented on lib/main.dart:
```diff
@@ -10,170 +11,83 @@ class MainApp extends StatelessWidget {
+                    const Text(
+                      'Flutter-Developer',
+                      style: TextStyle(
+                          fontFamily: 'Roboto', fontWeight: FontWeight.bold),
```
Давайте начнем делать заготовки для темы приложения. Выносим в отдельные классы. В папке lib делаем папку assets и там создаем папки colors(для цветов), res (для путей к картинкам), strings (для строк), text(для стилей текста). В них классы примерно такие:

```dart
/// Строковые константы приложения
class AppStrings {
  static const appTitle = 'Это тайтл';
}

class AppAssets {
  static const favorite = 'res/icons/heart_icon.svg';
}

class AppTypography {
  static const textText16Regular = TextStyle(
    fontSize: 16.0,
    height: 1.25,
    fontWeight: FontWeight.w400,
  );
}

class AppColors {
  static const myColor = Colors.red;
}
```

Evgenia-bit reviewed (changes requested):
Все хорошо 👍  
Есть только немного моментов в комментариях, которые нужно поправить.
Также есть правило помещать шрифты и картинки в корень проекта в директорию `assets/`, так что переместите пожалуйста папки `fonts/` и `images/` из `lib/` в `assets/`.
Помимо этого, просьба прикладывать видео или скриншот вашего приложения.

ArsenijDKS committed:
new file:   lib/assets/colors/appColors.dart
	new file:   lib/assets/res/AppAssets.dart
	new file:   lib/assets/strings/AppStrings.dart
	new file:   lib/assets/text/AppTypography.dart
	modified:   lib/main.dart
	modified:   pubspec.yaml
